src/main/java/com/easypost/model/Shipment.java

```diff
@@ -35,6 +35,7 @@
 	String uspsZone;
 	Map<String, Object> options;
 	List<ShipmentMessage> messages;
+    List<TaxIdentifier> taxIdentifiers;
```

Comment:
If this looks unindented to you, it's just a GitHub thing.

Will verify with future code styling cleanup.

Reply:
The indentation is also off if you view the raw file: https://github.com/EasyPost/easypost-java/blob/59a6fb059e8f4056e2f5b9444fe8728ca58f0e8f/src/main/java/com/easypost/model/Shipment.java

There's a mix of tabs and spaces being used in this file. The existing indentation character in this file appears to use `<tab>`, but I would prefer that we eventually move to using (4?) spaces.

Reply:
New indent_cleanup branch for this: https://github.com/EasyPost/easypost-java/tree/indent_cleanup